ext/opcache/Optimizer/zend_func_info.c

```diff
@@ -1015,7 +1015,7 @@ static const func_info_t func_infos[] = {
 	F0("mysqli_select_db",						MAY_BE_NULL | MAY_BE_FALSE | MAY_BE_TRUE),
 	F1("mysqli_sqlstate",						MAY_BE_NULL | MAY_BE_STRING),
 	F0("mysqli_ssl_set",						MAY_BE_NULL | MAY_BE_TRUE),
-	F1("mysqli_stat",							MAY_BE_NULL | MAY_BE_STRING),
+	F1("mysqli_stat",							MAY_BE_FALSE | MAY_BE_STRING),
```

Comment:
This will needs to include MAY_BE_NULL (on 7.4).

Reply:
Done